README.md

```diff
@@ -1,2 +1,239 @@
-# math-magicians-app
-"Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to make simple calculations and read a randomly generated math-related quote.
+# Math-Magician App: Ometman
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="./ometman-logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
![image](https://github.com/ometman/math-magicians-app/assets/93039359/48d17ee5-bde7-4704-a653-56741f86a618)

- [ ] It would be nice if you fix the logo link because it is broken and giving me an error which you can see in the image above ⬆️ Kindly fix your logo link or if you don't have any logo for your app right now then kindly remove this link 😊 Your readme should have no broken links because broken links make your readme not professional 😨

Reply:
Fixed. Thanks.